.github/workflows/e2e-linux.yaml

```
@@ -154,22 +154,22 @@ jobs:
     steps:
       - name: configure aws credentials
         uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
-        if: ${{ needs.test.outputs.vars == 'true' }} && ((startsWith(inputs.os, 'amazon') && inputs.version != '2' )) && github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
+        if: ${{ needs.test.outputs.vars == 'true' && ((startsWith(inputs.os, 'amazon') && inputs.version != '2' )) && github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true }}
```

A little more readable:
```suggestion
        if: >-
            ${{ needs.test.outputs.vars == 'true'
                && ((startsWith(inputs.os, 'amazon') && inputs.version != '2' ))
                && github.repository_owner == 'runfinch'
                && github.event.pull_request.user.login != 'dependabot[bot]'
            }}
```

Explanation:
`>-`: needed to make the statement work on multiple lines
`needs.test.outputs.vars`: We use the `vars` output of the `test` job (which redirects the `test` job's `has_creds` step -- we may want to make the names of the job's output and step output match)
`&& ((startsWith(inputs.os, 'amazon') && inputs.version != '2' ))`: skips this step when run on AL2
`&& github.repository_owner == 'runfinch'`: only runs when the repository_owner of the branch is `runfinch`. So, any PR made from the runfinch organization will be `true`, while any PR from an outside branch will be `false`
`&& github.event.pull_request.user.login != 'dependabot[bot]'`: disallows dependabot PRs from triggering this step explicitly, since they will be on the runfinch upstream